### PR TITLE
Docs: Add documentation on how to add the collapsible sections plugin to a Starlight project

### PR DESCRIPTION
**Pull request by jonasheschl**
Adding the collapsible sections plugin to a Starlight project works a bit differently from adding the plugin to a vanilla Astro project. Similar to the `Installation` page this PR adds additional documentation on how to add the plugin to a Starlight project.

### CODE REVIEW

File: docs/src/content/docs/plugins/collapsible-sections.mdx [hunk 1]
```diff
@@ -52,6 +54,50 @@ Before being able to use collapsible sections in your code blocks, you need to i
...
+      ```
+      </TabItem>
+      <TabItem label="yarn">
+      ```sh
+      yarn add @expressive-code/plugin-collapsible-sections
+      ```
+      </TabItem>
+    </Tabs>
+
+2. Add the integration to starlights configuration by adding it to the `plugins` list of the `expressiveCode` configuration object in your starlight configuration.
```
Comment (hippotastic) [inline docs/src/content/docs/plugins/collapsible-sections.mdx:81]:
```suggestion
2. Add the integration to Starlight's configuration by adding it to the `plugins` list of the `expressiveCode` configuration object in your Starlight configuration.
```

**Review by hippotastic — COMMENTED** [review]
Thank you very much for this PR!

I only made a minor suggestion (use proper capitalization of the term Starlight). Other than that, it looks good!

**Comment by jonasheschl** [comment]
> Thank you very much for this PR!
> 
> I only made a minor suggestion (use proper capitalization of the term Starlight). Other than that, it looks good!

Committed :+1:

**Review by hippotastic — APPROVED** [review]
LGTM!

**Pull request merged**